benchmark/FedHPOB/scripts/gcn/cora_dp.yaml

```diff
@@ -0,0 +1,41 @@
+use_gpu: True
+device: 0
+early_stop:
+  patience: 100
+seed: 12345
+federate:
```

Comment:
please make sure you don't need to inject noise in download channel. Without `join_in_info`, the server won't inject noise when the clients download the model